deb/common/control

```diff
@@ -61,7 +61,8 @@ Description: Docker: the open-source application container engine
 
 Package: docker-ce-cli
 Architecture: linux-any
-Depends: ${shlibs:Depends}
+Depends: docker-buildx-plugin,
+         ${shlibs:Depends}
 # TODO change once we support scan-plugin on other architectures
 Recommends: docker-scan-plugin [amd64]
```

Comment:
I'm still thinking if this should be `Recommends` to allow for "small" installations that are not used to build images (e.g. production situations). I realise the `buildx` plugin is required to perform `docker build`, but `docker run` (and anything else) can function without it. We already "control" that situation in the main CLI (produce an error if the component is missing when someone attempts to do a `docker build`); `Recommend` in itself is already a quite "strong" dependency, and by default will be installed. From [the handbook](https://debian-handbook.info/browse/stable/sect.package-meta-information.html#id-1.8.6.7.10.10);

> (...) the “recommended” dependencies, the most important, considerably improve the functionality offered by the package but are not indispensable to its operation.

And

> (...) You should always install the “recommended” packages, unless you know exactly why you do not need them. This is now also the default for APT unless configured otherwise.

Reply:
Yes agree, will put buildx that to Recommends instead.